src/redisearch_rs/rqe_iterators/src/lib.rs

```diff
@@ -31,13 +31,16 @@ pub enum RQEIteratorError {
     TimedOut,
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq)]
```

Comment:
approved, but why we cannot do `Eq`? Is there a reason why `RSIndexResult` cannot be `Eq`?